test/gtfs/hastus/trip_test.exs

```diff
@@ -0,0 +1,68 @@
+defmodule Gtfs.Hastus.TripTest do
+  use ExUnit.Case
+
+  alias Gtfs.Hastus.Trip
+
+  describe "parse" do
+    test "parses data" do
+      binary =
+        [
+          "schedule_id;area;run_id;block_id;start_time;end_time;start_place;end_place;route_id;trip_id",
+          "aba20021;123;    1501; 57 - 11;04:15;04:30;albny;wtryd;;   43858890",
+          "aba20021;123;    1501; 57 - 11;04:30;05:05;wtryd;hayms;  193;   43857919"
+        ]
+        |> Enum.join("\n")
+
+      assert Trip.parse(binary) == [
+               %Trip{
+                 schedule_id: "aba20021",
+                 run_id: "123-1501",
+                 block_id: "57-11",
+                 start_time: "04:15",
+                 end_time: "04:30",
+                 start_place: "albny",
+                 end_place: "wtryd",
+                 route_id: nil,
+                 trip_id: "43858890"
+               },
+               %Trip{
+                 schedule_id: "aba20021",
+                 run_id: "123-1501",
+                 block_id: "57-11",
+                 start_time: "04:30",
+                 end_time: "05:05",
+                 start_place: "wtryd",
+                 end_place: "hayms",
+                 route_id: "193",
+                 trip_id: "43857919"
+               }
+             ]
+    end
+
+    test "filters out rows with missing data" do
```

Comment:
This feels misleading if I understand the functionality properly. Some fields (e.g. `route_id`) can be missing, right?

Reply:
There are always 4 or 5 trips at the top of the file that are missing some data (schedule_id, area, run, route). I don't know why they exist, but this just tests that they don't break anything.

Would it be helpful to have a comment by the test or a different name that explains that situation, so it's clearly about those malformed rows rather than normally-missing route ids?

Reply:
If the dest description said "malformed rows" or "missing area code" or something I think that would be helpful—I took this to mean "rows with any missing data".